docs/self-hosted/config-options.md

```diff
@@ -236,8 +236,8 @@ prefixing the value with `{type}:`. The following types are available:
 | `LOG_LEVEL`                | What level of detail to log. One of `fatal`, `error`, `warn`, `info`, `debug`, `trace` or `silent`.        | `info`                       |
 | `LOG_STYLE`                | Render the logs human readable (pretty) or as JSON. One of `pretty`, `raw`.                                | `pretty`                     |
 | `MAX_PAYLOAD_SIZE`         | Controls the maximum request body size. Accepts number of bytes, or human readable string.                 | `1mb`                        |
-| `ROOT_REDIRECT`            | Where to redirect to when navigating to `/`. Accepts a relative path, absolute URL, or `false` to disable. | `./admin`                    |
-| `SERVE_APP`                | Whether or not to serve the Admin App under `/admin`.                                                      | `true`                       |
+| `ROOT_REDIRECT`            | Redirect the root of the application `/` to a specific route. Accepts a relative path, absolute URL, or `false` to disable. | `./admin`                    |
+| `SERVE_APP`                | A boolean that enables/disables your Directus application entirely.                                                      | `true`                       |
```

Comment:
I don't think this description makes it much clearer. Now it could be understood that the entire Directus application gets enabled/disabled by this option, while this only refers to the _serving of the web application_. Should we use the term _Data Studio_ here to make it more clear?

Reply:
I agree. I'd like to go with "Whether or not to serve the Admin Web Application"

Saying "entirely" is too much imo.